pmmdemo/percona_server_80.tf

```diff
@@ -45,12 +45,11 @@ resource "random_password" "mysql80_replica_password" {
 
 resource "random_password" "mysql80_sysbench_password" {
   length      = 30
-  special     = true
+  special     = false
```

Comment:
An exclamation sign was the reason for sysbench not being able to authenticate with mysql.